netbox/templates/extras/object_imageattachments.html

```diff
@@ -0,0 +1,35 @@
+{% extends base_template %}
+{% load i18n %}
+{% load render_table from django_tables2 %}
+{% load thumbnail %}
+
+{% block extra_controls %}
+  {% if perms.extras.add_imageattachment %}
+    <a href="{% url 'extras:imageattachment_add' %}?object_type={{ object|content_type_id }}&object_id={{ object.pk }}" class="btn btn-primary">
+      <span class="mdi mdi-plus-thick" aria-hidden="true"></span> {% trans "Attach an Image" %}
+    </a>
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+  <div class="d-flex flex-wrap">
+    {% for object in image_attachments %}
+      <div class="thumbnail m-2">
+        {% thumbnail object.image "200x200" crop="center" as tn %}
+          <a href="{{ object.get_absolute_url }}" class="d-block" title="{{ object.description }}">
+            <img
+              src="{{ tn.url }}"
```

Comment:
So, IIRC, we recently [put all images behind a `login_required` decorator](https://github.com/netbox-community/netbox/issues/17972).

However, even though these are fairly small images without a ton of detail, using this thumbnail URL as is makes it possible to see a representation of the image outside of being logged in.

Reply:
I'm pretty sure I was. But I will double check.